Updated README.md (#45)
ipinfo/go: README.md

Check all the data we have for your IP address [here](https://ipinfo.io/what-is-my-ip).

- [Getting Started](#getting-started)
	- [Installation](#installation)
	- [Quickstart](#quickstart)
- [Authentication](#authentication)

> - [Internationalization](#internationalization)

Reviewer: Can we also add `CountryFlag` here that was merged in the last commit?

Author: Hey, @rm-Umar, what would be the best way to go about this? Shall we merge this issue now, then I can make a second PR specifically for the `CoutnryFlag` section?

	"fmt"
	"log"
	"net"
	"github.com/ipinfo/go/v2/ipinfo"
)

func main() {
	const token = "YOUR_TOKEN"
	

> 	// params: httpClient, cache, token. You can set the first two params as `nil`

Reviewer: ```suggestion
	// params: httpClient, cache, token. `http.DefaultClient` and no cache will be used in case of `nil`.
```

Reviewer: We should specify what will be selected in case of nil as this statement might confuse some users.

	client := ipinfo.NewClient(nil, nil, token)

	const ip_address = "8.8.8.8"
	info, err := client.GetIPInfo(net.ParseIP(ip_address))

	if err != nil {
		log.Fatal(err)
	}

	fmt.Println(info)
	// Output: {8.8.8.8 dns.google false true Mountain View California US United States...
}
```

This data is available even on our free tier which includes upto 50,000 IP geolocation requests per month.

# Authentication

The IPinfo Go library ca be authenticated with your IPinfo API access token, which is passed as the third positional argument of the `ipinfo.NewClient()` method. Your IPInfo access token can be found in the account section of IPinfo's website after you have signed in : https://ipinfo.io/account/token

```go
const token = "YOUR_TOKEN"

>  // params: httpClient, cache, token. You can set the first two params as `nil`

Reviewer: Same as above

	}

	fmt.Println(result)
	// Ouptut: {100 100 map[CN:7 DE:4 JP:12 MX:3 US:32] map[Columbus, ...

}

```

# Batch Operations / Bulk Lookup

You can do batch lookup or bulk lookup quite easily as well. The inputs supported:

- IP addresses. IPV4 and IPV6 both
- ASN
- Specific field endpoint of an IP address e.g. `8.8.8.8/country`
   

```go
package main

import (
	"fmt"
	"log"
	"time"
	"github.com/ipinfo/go/v2/ipinfo"
	"github.com/ipinfo/go/v2/ipinfo/cache"
)

func main() {
	client := ipinfo.NewClient(
		nil,
		ipinfo.NewCache(cache.NewInMemory().WithExpiration(5*time.Minute)),
		"YOUR_TOKEN",
	)

	// batchResult will contain all the batch lookup data
	batchResult, err := client.GetBatch(
		[]string{
			"104.193.114.182",                    // you can pass IPV4 address
			"8.8.8.8/country",                    // you can get specific information
			"AS36811",                            // you can lookup ASN details
			"2a03:2880:f10a:83:face:b00c:0:25de", // IPV6 address
		},
		ipinfo.BatchReqOpts{
			BatchSize:       2,
			TimeoutPerBatch: 0,
			TimeoutTotal:    5,
		},
	)
	if err != nil {
		log.Fatal(err)
	}
	for k, v := range batchResult {
		fmt.Printf("k=%v v=%v\n", k, v)
	}

> 	fmt.Println()

Reviewer: ```suggestion

```
